src/components/NewLocationPage/LocationName/LocationName.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+import { materialSMBreakpoint } from 'assets/theme/sizes';
+
+export const RegionContainer = styled.div`
+  @media (max-width: ${materialSMBreakpoint}) {
```

Comment:
lets keep with the current format for breakpoints and use `min-width` instead of `max-width`. so:

```
export const RegionContainer = styled.div`
  text-align: center;
  @media (min-width: ${materialSMBreakpoint}) {
    text-align: left;
  }
```
`;